packages/office-ui-fabric-react/src/components/SpinButton/SpinButton.classNames.ts

```diff
@@ -20,10 +20,11 @@ export const getClassNames = memoizeFunction(
     disabled: boolean,
     isFocused: boolean,
     keyboardSpinDirection: KeyboardSpinDirection,
-    labelPosition: Position = Position.start
+    labelPosition: Position = Position.start,
+    className: string | undefined = undefined
```

Comment:
This classname should probably be `className?: string` - it does not match the interface otherwise (`className?: string` has different semantics from `classname: string | undefined`)